doc/man3/X509_LOOKUP_meth_new.pod

```diff
@@ -0,0 +1,165 @@
+=pod
+
+=head1 NAME
+
+X509_LOOKUP_meth_new, X509_LOOKUP_meth_free, X509_LOOKUP_meth_set_new_item,
+X509_LOOKUP_meth_get_new_item, X509_LOOKUP_meth_set_free,
+X509_LOOKUP_meth_get_free, X509_LOOKUP_meth_set_init,
+X509_LOOKUP_meth_get_init, X509_LOOKUP_meth_set_shutdown,
+X509_LOOKUP_meth_get_shutdown, X509_LOOKUP_meth_set_ctrl,
+X509_LOOKUP_meth_get_ctrl, X509_LOOKUP_meth_set_get_by_subject,
+X509_LOOKUP_meth_get_get_by_subject, X509_LOOKUP_meth_set_get_by_issuer_serial,
+X509_LOOKUP_meth_get_get_by_issuer_serial,
+X509_LOOKUP_meth_set_get_by_fingerprint,
+X509_LOOKUP_meth_get_get_by_fingerprint, X509_LOOKUP_meth_set_get_by_alias,
+X509_LOOKUP_meth_get_get_by_alias, X509_LOOKUP_set0_method_data, X509_LOOKUP_get0_method_data,
+X509_LOOKUP_get0_store, X509_OBJECT_set1_X509, X509_OBJECT_set1_X509_CRL,
```

Comment:
Remove the final "," and end with a "-" followed by a brief description of the purpose of these functions. See other pod files for examples